### PR TITLE
[android] cleanup default styles.xml and themes

### PR DESCRIPTION
**Pull request by jonathanpeppers**
### Description of Change ###

Going forward, default Maui apps on Android can simply use:

```csharp
[Activity(Theme = "@style/Maui.MainTheme", MainLauncher = true)]
public class MainActivity : MauiAppCompatActivity
{
}
```

You could also append `.NoActionBar` to remove the `ActionBar` if needed.

To help with migration from Xamarin.Forms, I left some "compatibility"
themes intact:

```xml
<!-- Themes for Xamarin.Forms backwards compatibility -->
<style name="MainTheme" parent="Maui.MainTheme" />
<style name="MainTheme.Base" parent="Maui.MainTheme" />
<style name="MainTheme.NoActionBar" parent="Maui.MainTheme.NoActionBar" />
<style name="AppTheme" parent="Maui.MainTheme" />
<style name="AppTheme.NoActionBar" parent="Maui.MainTheme.NoActionBar" />
```

These are used in many of the current Xamarin.Forms templates &
samples, so it will probably be helpful to leave these names in.

I deleted `styles.xml` and `colors.xml` where no longer needed in
tests, samples, and templates. We can use `Maui.MainTheme` instead.

Other unused things I deleted:

* `strings.xml` -> `action_settings`
* `strings.xml` -> `app_name`
* `dimens.xml` -> `fab_margin`

These must have been leftover from old Xamarin.Forms templates or
something. I couldn't find anything using them.

In a future PR, I will need add a new `Maui.SplashTheme` for splash
screen support for "single project".

### Additions made ###

* Adds `Maui.MainTheme` and `Maui.MainTheme.NoActionBar`.

### PR Checklist ###

- [ ] Targets the correct branch 
- [ ] Tests are passing (or failures are unrelated)
- [ ] Targets a single property for a single control (or intertwined few properties)
- [ ] Adds the property to the appropriate interface
- [ ] Avoids any changes not essential to the handler property
- [ ] Adds the mapping to the PropertyMapper in the handler
- [ ] Adds the mapping method to the Android, iOS, and Standard aspects of the handler
- [ ] Implements the actual property updates (usually in extension methods in the Platform section of Core)
- [ ] Tags ported renderer methods with [PortHandler]
- [ ] Adds an example of the property to the sample project (MainPage)
- [ ] Adds the property to the stub class
- [ ] Implements basic property tests in DeviceTests

### CODE REVIEW

File: src/Compatibility/Core/src/Android/Resources/values/styles.xml [hunk 1]
```diff
@@ -24,29 +24,4 @@
     <item name="scrollViewStyle">@style/scrollViewScrollBars</item>
   </style>
 
-  <style name="MainTheme" parent="MainTheme.Base">
-  </style>
-  
-  <!-- Base theme applied no matter what API -->
-  <style name="MainTheme.Base" parent="Theme.AppCompat.DayNight.DarkActionBar">
```
Comment (jonathanpeppers) [inline src/Compatibility/Core/src/Android/Resources/values/styles.xml:31]: Equivalents of these moved to `src/Core/src/Platform/Android/Resources/values/styles.xml`

File: src/Core/src/Platform/Android/Resources/values/strings.xml [hunk 2]
```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<resources>
-    <string name="app_name">Xamarin</string>
-    <string name="action_settings">Settings</string>
```
Comment (jonathanpeppers) [inline src/Core/src/Platform/Android/Resources/values/strings.xml:4]: I did not find anything that actually used these. Delete them?

**Comment by AmrAlSayed0** [comment]
We've actually ditched our Splash Screen instead we set the ```SplashScreen``` theme right on the MainActivity and immediately replace that with the ```MainTheme``` on ```OnCreate``` and it made things so much simpler with deeplinks and app actions and such so the ```Maui.SplashTheme``` may not be strictly necessary.

**Comment by jonathanpeppers** [comment]
`Maui.SplashTheme` will be for a future feature, so you can do something like:

```xml
<MauiSplashScreen Include="someimage.svg" Color="#FF69B4" />
```

You won't have to use this, if your app doesn't need it. But we will need a theme for it.

File: src/Core/src/Platform/Android/Resources/values/strings.xml (hunk 2 above)
Comment (PureWeen) [reply src/Core/src/Platform/Android/Resources/values/strings.xml:4]: It looks like that was part of the .NET MAUI samples we were originally using

https://github.com/dotnet/maui/commit/3aa86b4c988355203998fb9550dd85d7c266ad38#diff-7c01aad57ec768e8b14c209f0ddfbe8ed735c8741698e3b11bac8cf71d65d145R3

I don't see a reference to this at all over on the XF repository so I'd say fine to delete

**Review by PureWeen — APPROVED** [review]

**Review by Matt89James — COMMENTED** [review]
Do you have the codes to find a wireless device through the html program?

**Pull request merged**